lib/block.js

```diff
@@ -11,8 +11,8 @@ function Block(options){
 
 Block.prototype.coordinates = function () {
   var coordinates = []
-  for(x = this.x; x <= (this.x + this.width); x++){
-    for(y = this.y; y <= (this.y + this.height); y++){
+  for( var x = this.x; x <= (this.x + this.width); x++){
```

Comment:
Great change. Glad she went over that this morning, I didn't remember the importance of this.